tests/test_ilastik/test_applets/dataSelection/testDataSelectionGui.py

```diff
@@ -80,9 +80,12 @@ def test_scale_select_exists_and_triggers_gui_event(dataset_table, mock_gui):
 def test_locked_scale_select_does_not_trigger_gui_and_informs_user(dataset_table, mock_gui, intercept_info_popup):
     dataset_table.scaleSelected.connect(mock_gui.handleScaleSelected)
     scale_cell_multiscale_locked = dataset_table.model().index(2, DatasetColumn.Scale)
-    editor_locked = dataset_table.indexWidget(scale_cell_multiscale_locked)
+    editor = dataset_table.indexWidget(scale_cell_multiscale_locked)
```

Comment:
at the start of the file
```
from PyQt5.QtWidgets import QComboBox, QMessageBox, QApplication
```
plus
```suggestion
    editor = dataset_table.indexWidget(scale_cell_multiscale_locked)
    QApplication.processEvents()
```

makes it work for me

Reply:
Nice, thanks. Let's see if it works.

Reply:
Looks promising, all green on the first try. I restarted macos-ilastik just to give it another run, since the problem before was flaky, not 100% fail (though tbh, it was failing more often than not).